lib/migrations/2_deploy_organization.ts

```diff
@@ -110,12 +110,25 @@ export const arcJsDeployer = (web3: Web3, artifacts: any, deployer: any): void =
 
     const AvatarInst = await Avatar.at(tx.logs[0].args._avatar);
     const nativeTokenAddress = await AvatarInst.nativeToken();
+    let genTokenAddress;
+
+    if (network !== "live") {
+      // create a global GEN token
+      // await deployer.deploy(DaoToken, tokenName, tokenSymbol, 0);
+      // genTokenAddress = (await DaoToken.deployed()).address;
+      genTokenAddress = nativeTokenAddress;
+      console.log(`using global GEN token for network != "live" at: ${genTokenAddress}`);
+    } else {
+      // the "real" live ETH GEN token
+      genTokenAddress = "0x543Ff227F64Aa17eA132Bf9886cAb5DB55DCAddf";
```

Comment:
how come this address isn't the Avatar's nativeToken()? shouldn't it be?

Reply:
it is? how is it saying that? it seems to me that GEN should always be the native token of the Genesis DAO when deployed on any network

Reply:
this gets passed to the constructor of `GenesisProtocol` which accepts the staking token.

Reply:
https://github.com/daostack/arc/blob/master/contracts/VotingMachines/GenesisProtocol.sol#L96